source/integrations/jira.rst

```diff
@@ -3,6 +3,6 @@
 Jira Plugin 
 ================================
 
-Documentation has moved to `Jira plugin repository <https://github.com/mattermost/mattermost-plugin-jira>`_.
+The Jira/Mattermost plugin documentation is currently being updated and relocated to a new location: https://mattermost.gitbook.io/jira-plugin/
 
 Learn more about how to integrate with your other `Atlassian tools <https://docs.mattermost.com/deployment/atlassian-integrations.html>`.
```

Comment:
Not part of this PR but the link formatting here is not correct.

Reply:
woudl this be better?